src/components/MoveList.js

```diff
@@ -1,30 +1,72 @@
+import { useState } from 'react'
 import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import youTubeIcon from '../assets/img/youTube.png'
+import Video from '../components/Video'
 
 MoveList.propTypes = {
   moves: PropTypes.array.isRequired,
+  toggleVideo: PropTypes.func,
 }
 
 export default function MoveList({ moves }) {
-  const listItems = moves.map(({ name, id }) => <li key={id}>{name}</li>)
+  const [video, setVideo] = useState({})
+
+  const listItems = moves.map(({ name, id, video_id, video_start }) => (
+    <li key={id}>
+      <button
+        data-testid="listitem-button"
+        onClick={() => openVideo(video_id, video_start)}
```

Comment:
You could could directly use `setVideo`  and remove `openVideo`

```suggestion
        onClick={() => setVideo(video_id, video_start)}
```

Reply:
Yeah... why not?. Guess I'll it this way